Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoApi.kt

```diff
@@ -0,0 +1,185 @@
+package com.braintreepayments.api.venmo
+
+import android.text.TextUtils
+import com.braintreepayments.api.core.ApiClient
+import com.braintreepayments.api.core.BraintreeClient
+import com.braintreepayments.api.core.BraintreeException
+import com.braintreepayments.api.core.MetadataBuilder
+import com.braintreepayments.api.venmo.VenmoAccountNonce.Companion.fromJSON
+import org.json.JSONArray
+import org.json.JSONException
+import org.json.JSONObject
+
+internal class VenmoApi(
+    private val braintreeClient: BraintreeClient,
+    private val apiClient: ApiClient
+) {
+
+    fun createPaymentContext(
+        request: VenmoRequest,
+        venmoProfileId: String?,
+        callback: VenmoApiCallback
+    ) {
+        val params = JSONObject()
+        try {
+            params.put("query", "mutation CreateVenmoPaymentContext(\$input: " +
+                        "CreateVenmoPaymentContextInput!) { createVenmoPaymentContext" +
+                        "(input: \$input) { venmoPaymentContext { id } } }"
```

Comment:
We could use a kotlin multiline string here to properly format the graphql query: https://kotlinlang.org/docs/java-to-kotlin-idioms-strings.html#use-multiline-strings